packages/data-generator/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@olo/data-generator",
```

Comment:
This will fail the publish step since we don't use `@olo` on public npm. Maybe `olo-data-generator`?

Reply:
Addressed in rebased 5c23ce5